### PR TITLE
Bugfix 705/check mode debian

### PR DESCRIPTION
**Pull request by piotrkochan**
#705

### CODE REVIEW

File: roles/zabbix_agent/tasks/Debian.yml [hunk 1]
```diff
@@ -199,8 +199,7 @@
   when: ansible_distribution in ['Ubuntu', 'Debian']
   register: zabbix_agent_package_installed
   until: zabbix_agent_package_installed is succeeded
-  become: true
-  check_mode: false
+  become: yes
```
Comment (brnl) [inline roles/zabbix_agent/tasks/Debian.yml:202]: Why did you change this to `yes` while `true`|`false` is used everywhere in this project? 🤔

**Comment by brnl** [comment]
Note: The issue also exists on line 227: [`roles/zabbix_agent/tasks/Debian.yml`: 227 "Debian | Installing zabbix-{sender,get}"](https://github.com/ansible-collections/community.zabbix/blob/2287bc4165/roles/zabbix_agent/tasks/Debian.yml#L227)

**Review by D3DeFi — APPROVED** [review]
@piotrkochan thanks for the patch!

@brnl thank you for helping with review :)
